databases/week1/hyfHomeworkWeek1.sql

```diff
@@ -0,0 +1,91 @@
+--Dropped old databse because, few entries were deleted or modified.
+DROP DATABASE hyfHomeworkWeek1;
+
+--Created new database.
+CREATE DATABASE  hyfHomeworkWeek1;
+USE hyfHomeworkWeek1;
+
+-- Retrieving tables for detailed reference
+USE hyfHomeworkWeek1;
+SELECT * FROM user;
+SELECT * FROM task;
+SELECT * FROM status;
+
+
+--1. Select the names and phones of all users;
+USE hyfHomeworkWeek1;
```

Comment:
You don't need to repeat USE command for each query - once in the very beginning is enough. ☺️